packages/@react-spectrum/label/src/ReadOnlyField.tsx

```diff
@@ -48,34 +49,36 @@ function ReadOnlyField(props: SpectrumFieldProps, ref: RefObject<HTMLTextAreaEle
   }, [onHeightChange, readOnlyText, ref]);
 
   return (
-    <div
-      className={
-        classNames(
-          styles,
-          'spectrum-Textfield',
-          {
-            'spectrum-Textfield--quiet': true,
-            'spectrum-Textfield--readonly': true
-          }
-        )
-      }>
-      <FocusRing focusRingClass={classNames(styles, 'focus-ring')} isTextInput autoFocus={autoFocus}>        
-        <textarea
-          {...mergeProps(inputProps, hoverProps)} 
-          rows={1}
-          ref={ref}
-          value={readOnlyText}
-          className={
-            classNames(
-              styles,
-              'spectrum-Textfield-input',
-              {
-                'is-hovered': isHovered
-              }
-            )
-        } />
-      </FocusRing> 
-    </div>
+    <Flex direction="column" UNSAFE_className={classNames(labelStyles, 'spectrum-Field-wrapper')}>
```

Comment:
This should not be needed anymore due to https://github.com/adobe/react-spectrum/pull/3277/files/bcf277811b211d4767fe1f5ddd912213156b9b76..bd3ae4aaba0366290ca21fcb672a53b9da3cec47#diff-abb1c606574077fc353b2ee07124e2a936b3a7a32ae8fe4ff471b65be704887dR105